routes/login.tsx

```diff
@@ -173,13 +173,19 @@ export default function Login({ data }: PageProps<Data, FreshContextState>) {
           </strong>.
         </p>
 
-        <h2 class='text-2xl mb-4 text-center'>Need help?</h2>
-        <p class='text-center mt-2 mb-6'>
-          If you're having any issues or have any questions,{' '}
-          <strong>
-            <a href={`mailto:${helpEmail}`}>please reach out</a>
-          </strong>.
-        </p>
+        {helpEmail !== ''
+          ? (
+            <section>
```

Comment:
You shouldn't need a section here, I think you can just do a fragment, i.e. `<>` and `</>` and avoid this extra DOM section/nesting.

Reply:
I don't use preact/react normally and I forgot about fragments, ty. I have updated the code.